### ISSUE TITLE
Group Number Change

### ISSUE THREAD

**Issue by zpellish** [MarchingLizard/brifony#35]
Adam,

Birthright changed the group number for one of our flights, but kept the flight details exactly the same. I can't edit the SH-34-xxx in the flight table in BriReg, and I'm worried if I delete this flight it will  kick everyone off who has already been assigned. Can you change SH-34-522 to SH-34-911 without it impacting the applicants already assigned to that group?

**Comment by IshaDakota** [comment]
@zpellish yeah, that's a case we didn't anticipate. I think we may have to reassign them to the new flight. I'll check.
